src/com/google/enterprise/adaptor/database/MetadataColumns.java

```diff
@@ -62,13 +60,14 @@
       log.fine("element: `" + e + "'");
       e = e.trim(); 
       String def[] = e.split(":", 2);
-      if (2 != def.length) {
-        String errmsg = "expected two parts separated by colon: `" + e + "'";
-        throw new InvalidConfigurationException(errmsg);
+      if (def.length == 1) {
+        String columnName = def[0].trim();
```

Comment:
We know there's at least one (possibly empty) entry in def, so you could lift these lines out of this if statement.